FindTeamer/Models/EventModel.swift

```diff
@@ -6,15 +6,28 @@
 //
 
 import Foundation
-import FirebaseFirestoreSwift
 
-class EventModel: Identifiable, Codable {
-    @DocumentID var id: String?
-    var eventType = ""
-    var eventTitle = ""
-    var eventEmail = ""
-    var eventPhoneNumber = ""
-    var eventAddress = ""
-    var eventTime: Date = Date()
-    var eventDescription = ""
+struct EventModel: Identifiable {
+    // MARK: Fields
+    var id: String = UUID().uuidString
+    var eventType: String
+    var eventTitle : String
+    var eventEmail: String
+    var eventPhoneNumber: String
+    var eventAddress: String
+    var eventTime: Date
+    var eventDescription: String
```

Comment:
```suggestion
    var eventDescription: String?
```
It can be optional rather than empty during initialization if based on business rules it's can be empty